docs/index.asciidoc

```diff
@@ -393,6 +405,25 @@ If you don't set a value for this option:
 - for elasticsearch clusters 6.x: the value of 'doc' will be used;
 - for elasticsearch clusters 5.x and below: the event's 'type' field will be used, if the field is not present the value of 'doc' will be used.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: does not provide ECS-compatible templates
+** `v1`: provides defaults that are compatible with v1 of the Elastic Common Schema
+* Default value depends on which version of Logstash is running:
+** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+** Otherwise, the default value is `disabled`.
+
+Controls this plugin's desired compatibility with the https://www.elastic.co/blog/introducing-the-elastic-common-schema[Elastic Common Schema],
```

Comment:
```suggestion
Controls this plugin's compatibility with the https://www.elastic.co/blog/introducing-the-elastic-common-schema[Elastic Common Schema],
```

Reply:
If you'd like to link to the ECS documentation (in addition or instead of), here's the formatting you need:
`{ecs-ref}[Elastic Common Schema (ECS)]`

Reply:
heyyy. perfect.